plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduLatestConnectorTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kudu;
+
+import io.trino.testing.QueryRunner;
+import io.trino.tpch.TpchTable;
+
+import java.util.Optional;
+
+import static io.trino.plugin.kudu.KuduQueryRunnerFactory.createKuduQueryRunnerTpch;
+
+public class TestKuduLatestConnectorTest
+        extends BaseKuduConnectorTest
```

Comment:
Should this be a ConnectorSmokeTest only?

Reply:
I've been a bit out of the loop on the changes from Distributed/Smoke tests to these `BaseConnectorTest` and `BaseConnectorSmokeTest`. I'm okay keeping the "latest" tests as a smoke test if that has been the pattern that has been adopted for testing the later versions of the software. Makes sense since the expectation is that newer versions should work and we don't need to go through the full breadth of testing and a quick smoke test is sufficient.

Reply:
that's my thinking as well